docs/conf.py

```diff
@@ -54,9 +54,11 @@
     "sphinx_autodoc_typehints",
     "sphinx.ext.intersphinx",
     "sphinx.ext.viewcode",
+    "sphinx.ext.autosectionlabel",
 ]
 autodoc_default_flags = ["inherited-members", "members"]
 autodoc_member_order = "bysource"
+autosectionlabel_prefix_document = True
```

Comment:
reference labels are `path/to/file:My Section Title` (instead of the default, which is just `My Section Title` and not namespaced by file)